core/src/main/res/values/strings.xml

```diff
@@ -861,4 +860,4 @@
     <string name="on_demand_config_setting_changed">Setting updated successfully.</string>
     <string name="on_demand_config_stream_text">Looks like you stream a lot. Do you want episode lists to show stream buttons?</string>
     <string name="on_demand_config_download_text">Looks like you download a lot. Do you want episode lists to show download buttons?</string>
-</resources>
+</resources>
```

Comment:
Please revert the changes to the strings file, so that it looks like it was before. Small changes like the ones you added cause merge conflicts and clutter the git history. Apart from that, it looks good :+1: